### DISCUSSION TITLE
2023.4.0

### DISCUSSION THREAD

**Discussion by PeteRager** [PeteRager/lennoxs30#232] [Announcements]
# Version 2023.4.0 - S40 remote sensor support, Maintenance release

Requires HASS >= 2023.3.0, tested on 2023.4.6

## S40 Remote Sensor Support

S40 controllers are being deployed -  this release provides support for the new remote BLE sensors.  Docs are here:
https://github.com/PeteRager/lennoxs30#s40-remote-sensors

Thanks to @ghwong for working with me and providing diagnostic message logs!

##  Maintenance HASS deprecations

The Alert Sensor reports the alert string and was marked as "state increasing" which is invalid for text based data.  This state class has been removed.

Unit of Measures were using old deprecated constants; which HASS has replaced with Enum.  The code has been updated.

## What's Changed
* S40 Bluetooth Sensors by @PeteRager in https://github.com/PeteRager/lennoxs30/pull/230

**Full Changelog**: https://github.com/PeteRager/lennoxs30/compare/2023.1.0...2023.4.0

<hr /><em>This discussion was created from the release <a href='https://github.com/PeteRager/lennoxs30/releases/tag/2023.4.0'>2023.4.0</a>.</em>

**Comment by ghwong** [comment]
Here are some of the reading from my two remote sensors. Have been logging it since the new release but unable to figure out the different between analog, digital and temp reading from each sensors

![image](https://user-images.githubusercontent.com/10466928/234940151-7a3b5731-beb8-470b-84df-6662bc5126da.png)
